survey/1-profile.yml

```diff
@@ -35,7 +35,7 @@ questions:
       - US & Canada
       - Middle East
       - Others
-  - label: What is your current role?
+  - label: What is your current profile?
```

Comment:
Profile is just French for Role :)

I suggest the following:

- Keep "what is your current role?" in the 1-profile.yaml file
- Change "What is your role?" to "what is your occupation?" in the 2-work.yaml file

wdyt ?

Reply:
I would recommend using "What is your employment status?" in the work question  and keep "What your current role?" for the profile but we need to remove  `Founder or entrepreneur ` from work and add it to profile 
 @AbderrahimSoubaiElidrissi  @boredabdel  wdyt?

Reply:
@boredabdel @yjose Both solutions make sense but I think `occupation` makes more sense in the context of work and yes `Employment status` is a good description for the work (removing founder/entrepreneur) So I suggest:

- Keep "what is your occupation?" in the 1-profile.yaml file
- Keep "What is your employment status?" in the 2-work.yaml file